apps/host/src/components/component-loader/ComponentLoader.tsx

```diff
@@ -1,37 +0,0 @@
-import { component$, useClientEffect$, useSignal } from '@builder.io/qwik';
-import { RemoteData } from '../../../../../libs/shared/remotes';
-
-export interface Props {
-	remote: RemoteData;
-}
-
-export default component$<Props>(({ remote }) => {
-	const elementRef = useSignal<Element>();
-
-	useClientEffect$(() => {
-		const observer = new IntersectionObserver(
-			async ([element]) => {
-				if (element.isIntersecting) {
-					const response = await fetch(`/${remote.name}/`);
-					if (response.ok) {
-						const rawHtml = await response.text();
-						elementRef.value!.innerHTML = rawHtml;
-						observer.disconnect();
-					}
-				}
-			},
-			{ rootMargin: '150px' },
-		);
-		observer.observe(elementRef.value!);
-		return () => {
-			observer.unobserve(elementRef.value!);
-		};
-	});
-
-	return (
-		<div class="remote-component">
-			<p class="remote-label">{remote.url}</p>
```

Comment:
Note: the label here wasn't a link, my changes are fixing it

anyways I think it is quite beneficial to have a single remote component and re-use what we can instead of having code duplication which can lead to this type of bugs.